playground/frontend/integration_test/standalone_toggle_dark_mode_test.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+import 'package:playground_components_dev/playground_components_dev.dart';
+
+import 'common/common.dart';
+
+void main() {
+  IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+
+  testWidgets(
+    'Change dark mode',
+    (WidgetTester wt) async {
+      await init(wt);
+
+      expect(find.toggleThemeButton(), findsOneWidget);
+
+      Brightness getBrightness() {
+        return Theme.of(wt.element(find.toggleThemeButton())).brightness;
+      }
+
+      Future<void> toggleTheme() async {
+        await wt.tap(find.toggleThemeButton());
+        await wt.pumpAndSettle();
+      }
+
+      expect(getBrightness(), Brightness.light);
```

Comment:
Check the initial brightness, then test that it changed.

Reply:
done